Automattic/wp-calypso
# Update/move traffic to marketing

#### Changes proposed in this Pull Request

* Moves Seo related tools from Settings->Traffic to Marketing->traffic

#### Testing instructions

* Do this with a Jetpack site, Atomic site, free plan and premium or business.
* Goto /marketing/traffic/{site}
* Verify that you can use all the features on the page
* Goto /marketing/traffic/ does this take you to the sites selector?
* Goto: settings/traffic Does it take you to site selector then marketing/traffic
* Goto settings/traffic/{site} does it redirect to /marketing/traffic/{site}
